api/services/ocean/spark/paths/jobConfig.yaml

```diff
@@ -0,0 +1,35 @@
+put:
+  summary: "Put Job Config Template"
+  description: >
+    Attach or update an attached config template by replacing the existing configuration with the one provided in the request body.
+  operationId: "OceanSparkJobConfig"
+  tags:
+    - "Ocean Spark"
+  parameters:
+    - $ref: "../../../../commons/parameters/nonRequiredAccountId.yaml"
+    - $ref: "../parameters/path/oceanSparkClusterId.yaml"
+    - $ref: "../parameters/path/oceanSparkJobId.yaml"
+  requestBody:
+    required: false # if empty then the config is deleted 
+    content:
+      application/json:
+        schema:
+          type: object
+          properties:
+            displayName:
```

Comment:
Looking at the model used by the router, I don't think you can pass a `displayName` in the request body